FinniversKit/Sources/Components/SelectionListView/Subviews/SelectionListItemView.swift

```diff
@@ -94,12 +96,15 @@ class SelectionListItemView: UIView {
             detailViewsStackViewBottomConstraint.constant = -.spacingM
         }
 
-        textStackView.addArrangedSubviews([titleLabel, descriptionLabel])
+        if case .none = model.description {
```

Comment:
Nitpick: The titleLabel is always added so the if can just be for the descriptionLabel.